frontend/src/metabase/visualizations/visualizations/RowChart/utils/events.ts

```diff
@@ -66,8 +70,7 @@ const getColumnData = (columns: ColumnDescriptor[], datum: GroupedDatum) => {
 
         value = formatNullable(metricSum);
       } else {
-        const distinctValues = new Set(datum.rawRows.map(row => row[index]));
-        value = distinctValues.size === 1 ? datum.rawRows[0][index] : null;
+        value = datum.rawRows[seriesIndex][index];
```

Comment:
I don't understand the purpose of counting unique values here. It's been introduced in #26547.
@alxnddr do you remember what was it for?

Reply:
@kamilmielnik it was there to prevent the invalid behaivor. 
Let's add another row to the query:
```
SELECT
  'group_2',
  'sub_group_1',
  400,
  'something else'
```

Now the bar with `Y=group_2` and `Breakout=sub_group_1` represents two rows where `group_name` is `{ "group_2_sub_group_1", "something else" }` but with the fix applied we would extrapolate the URL with `group_2_sub_group_1` which is not correct.
However, I'm not agains this fix because it sort of makes the behavior consistent to other charts with breakouts which historically has been like this and even covered with specs AFAIR. 
Let's ask for @cdeweyx opinion, maybe it would be better to implement the correct behavior right away.